src/date/expand-pattern.js

```diff
@@ -33,10 +55,65 @@ return function( pattern, cldr ) {
 
 	switch ( true ) {
 		case "skeleton" in pattern:
-			result = cldr.main([
-				"dates/calendars/gregorian/dateTimeFormats/availableFormats",
-				pattern.skeleton
-			]);
+			flagDate = false, flagTime = false;
+			skeleton = pattern.skeleton;
+			if ( /[hHms]/.test( skeleton ) ) {
+				flagTime = true;
+			}
+			if ( /[GyQMd]/.test( skeleton ) ) {
+				flagDate = true;
+			}
+			if ( /Ed/g.test( skeleton ) ) {
+				flagDate = true;
+			} else if ( /Eh/g.test( skeleton ) || /EH/g.test( skeleton ) ||
+					/E/g.test( skeleton ) ) {
+				flagTime = true;
+			}
```

Comment:
Why are you testing these various combinations of `E`? `E` means day of week. Therefore, it's a date pattern. `d` means day, therefore it's a date pattern. `h` or `H` means hours, therefore it's a time pattern.